commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -98,19 +98,22 @@ def fulfill_order_placed_message_signal_task(
     canvas_entry_properties = {"products": []}
     canvas_entry_properties.update(extract_ct_order_information_for_braze_canvas(customer, order))
 
+    logger.info(
```

Comment:
I am not sure if we should do it this way since: 
This task is triggered from the fulfill endpoint which is called when the lineItem state of an item in the order is updated to fulfillment pending state.
What that means is we would be triggering a number of duplicate calls for fulfilling all the lineItems.
We might need to change our workflow and think about this perspective